components/mpas-ocean/bld/build-namelist

```diff
@@ -725,12 +725,22 @@ add_default($nl, 'config_sgr_salinity_prescribed');
 # Namelist group: coupling #
 ############################
 
-add_default($nl, 'config_remove_ais_river_runoff');
-if (($OCN_ICEBERG eq 'true') && ($OCN_FORCING eq 'active_atm')) {
+# Assume ice runoff corresponds to icebergs in Southern Ocean (true for JRA
+# forcing, perhaps modeled firn ice runoff approximates icebergs).
```

Comment:
Isn't the point here not so much that we assume *all* of the ice runoff corresponds to icebergs (true only for G-cases, not B-cases) but rather that we assume that anytime we are representing icebergs another way (presently DIB), we want to remove ice runoff completely?

Reply:
I was intending it more in an approximate sense, but maybe the comment is more. misleading than informative. Does it work for you to just remove my assumption:

```suggestion
# When we have a data representation of icebergs, remove ice runoff
```

Reply:
Okay, yeah, I'd like to have modification you suggested.  I think it helps clarify things.